backend/routes/api/auth/login.js

```diff
@@ -18,15 +18,27 @@ router.get(
 );
 
 router.get("/hello", auth.isAuthenticated, function(req, res) {
```

Comment:
We can delete this endpoint completely since it leaks sensitive information (`req.user` has OAuth ID)